docs/config.md

```diff
@@ -86,9 +86,8 @@ profile_models: <comma-delimited-string-list>
 # Duration of waiting time between each metric measurement in seconds
 [ monitoring_interval: <float> | default: 1 ]
 
-# Specifies whether to disable cpu monitor or not.
-# "auto" will disable cpu monitor when GPU(s) are used for profiling.
-[ cpu_monitor_disable: ("auto"|True|False) | default: "auto" ]
+# Specifies which metric(s) are to be collected.
+[ metrics: comma-delimited-list-string | default: `all non CPU metrics` ]
```

Comment:
Set the default here to say `default: See Config Defaults section`. Look at the argument `server_output_fields` for example.

Reply:
Updated.